tasks/main.yml

```diff
@@ -26,6 +26,16 @@
   register: autoclean_output
   changed_when: "'Del' in autoclean_output.stdout"
 
+- name: APT | Check for cached .deb files
+  shell: "ls /var/cache/apt/archives/*.deb 2&> /dev/null | wc -l"
```

Comment:
Your directing the output into null, then piping it into `wc`, it will always be 0..

```bash
$ ls /var/cache/apt/archives/*.deb 2&> /dev/null | wc -l
0
$ ls /var/cache/apt/archives/*.deb | wc -l
294
```